__tests__/app.test.js

```diff
@@ -92,3 +92,46 @@ describe("GET /api/articles/:article_id", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id/comments", () => {
+  test("responds with 200 and corresonding comment array", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body: { comments } }) => {
+        expect(comments).toBeInstanceOf(Array);
```

Comment:
We should not check if it is an array because we are using forEach which means it is 🙂

Reply:
Again this check on line 107 is redundant